common/src/main/java/io/netty/util/internal/InternalThreadLocalMap.java

```diff
@@ -41,18 +41,10 @@
 
     public static InternalThreadLocalMap getIfSet() {
         Thread thread = Thread.currentThread();
-        InternalThreadLocalMap threadLocalMap;
         if (thread instanceof FastThreadLocalThread) {
-            threadLocalMap = ((FastThreadLocalThread) thread).threadLocalMap();
-        } else {
-            ThreadLocal<InternalThreadLocalMap> slowThreadLocalMap = UnpaddedInternalThreadLocalMap.slowThreadLocalMap;
-            if (slowThreadLocalMap == null) {
-                threadLocalMap = null;
-            } else {
-                threadLocalMap = slowThreadLocalMap.get();
-            }
+            return ((FastThreadLocalThread) thread).threadLocalMap();
         }
-        return threadLocalMap;
+        return slowThreadLocalMap.get();
```

Comment:
So all good?

> Am 23.03.2016 um 03:14 schrieb Trustin Lee <notifications@github.com>:
> 
> In common/src/main/java/io/netty/util/internal/InternalThreadLocalMap.java:
> 
> >          }
> > -        return threadLocalMap;
> > +        return slowThreadLocalMap.get();
> Indeed, because we did not override initialValue(). LGTM
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>